hastory-cli/src/Hastory/Cli/OptParse/Types.hs

```diff
@@ -46,6 +50,7 @@ newtype ListRecentDirSets = ListRecentDirSets
     { lrdSetBypassCache :: Bool
     } deriving (Show, Eq)
 
-newtype Settings = Settings
-    { setCacheDir :: Path Abs Dir
-    } deriving (Show, Eq)
+data Settings = Settings
+    { setCacheDir         :: Path Abs Dir
+    , remoteStorageClient :: Maybe HastoryClient
+    } deriving (Show)
```

Comment:
Agreed. I did it to somewhat have one reusable http manager but it doesn't help since CLI commands are one-off anyway. Also storing a http manager in Settings is surely a bad design. Thanks!